pilot/pkg/xds/auth.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var AuthPlaintext = env.RegisterBoolVar("XDS_AUTH_PLAINTEXT", false,
-	"Authenticate plain text requests - used if Istiod is behind a gateway handling TLS").Get()
+	"Authenticate plain text requests - used if Istiod is behind a gateway or proxy handling TLS").Get()
 
```

Comment:
just to double check - this is unrelated to your xfcc changes or did you forget to check it?  I don't see the `AuthPlaintext` var called in your other changes

Reply:
This is unrelated. I just updated the description of the flag.

Reply:
so the incoming xfcc authentication logic doesn't need a feature flag or proxy version check?

Reply:
If you are updating this - please change it to 'if Istiod is running on a secure/trusted network'. 

"or proxy handling TLS' is not relevant here - plaintext is about not doing mTLS because the low-level network provide equivalent guarantees, having a proxy in front handling mTLS is an issue of authorizing the proxy.